moto/config/models.py

```diff
@@ -1110,6 +1142,134 @@ def put_evaluations(self, evaluations=None, result_token=None, test_mode=False):
             "FailedEvaluations": [],
         }  # At this time, moto is not adding failed evaluations.
 
+    def put_organization_conformance_pack(
+        self,
+        region,
+        name,
+        template_s3_uri,
+        template_body,
+        delivery_s3_bucket,
+        delivery_s3_key_prefix,
+        input_parameters,
+        excluded_accounts,
+    ):
+        # a real validation of the content of the template is missing at the moment
+        if not template_s3_uri and not template_body:
+            raise ValidationException("Template body is invalid")
+
+        if not re.match(r"s3://.*", template_s3_uri):
+            raise ValidationException(
+                "1 validation error detected: "
+                "Value '{}' at 'templateS3Uri' failed to satisfy constraint: "
+                "Member must satisfy regular expression pattern: "
+                "s3://.*".format(template_s3_uri)
+            )
+
+        pack = self.organization_conformance_packs.get(name)
+
+        if pack:
+            pack.conformance_pack_input_parameters = input_parameters
+            pack.delivery_s3_bucket = delivery_s3_bucket
+            pack.delivery_s3_key_prefix = delivery_s3_key_prefix
+            pack.excluded_accounts = excluded_accounts
+            pack.last_update_time = datetime2int(datetime.utcnow())
+            pack._status = "UPDATE_SUCCESSFUL"
```

Comment:
Can we haven an update()-method in the OrganisationConformancePack, to handle the value update?
The model itself should really be responsible for updating `last_update_time` and `status`

Reply:
done